### DISCUSSION TITLE
Quick fix list not checked by default in some cases

### DISCUSSION THREAD

**Discussion by stephenhawk8054** [uBlockOrigin/uBlock-issues#2744] [General]
There were 2 occasions I see that the quick fix list is not in default when checking troubleshooting information. The problem is it does not appear in neither `removed` nor `default` section

---

1. https://github.com/uBlockOrigin/uAssets/issues/19085#issuecomment-1642398620
2. OP tried to uninstall and reinstall uBO but the issue still occurs (YouTube gets ads)
https://www.reddit.com/r/uBlockOrigin/comments/154vtwy/getting_ads_on_youtube/jsr2458/

[OP's troubleshooting information](https://preview.redd.it/s61f3s5qh5db1.png?width=1250&format=png&auto=webp&s=8529b63179034efd32f6018964f5217602c32042)

In the 2nd case, I told OP to reinstall again and screenshot "My filters" tab, and what I see is [the quick fix list is unchecked by default](https://preview.redd.it/cd5wusfej5db1.png?width=1074&format=png&auto=webp&s=1d8892b9020c8c8b9911b7f525b53aab0f7ec7f5)

@gorhill Sorry do you know any issues similar to this before?

---

Some other users with same issue at the same reddit thread (quick fix list is unchecked): 

- https://www.reddit.com/r/uBlockOrigin/comments/154vtwy/getting_ads_on_youtube/jsr5xu0/

- https://www.reddit.com/r/uBlockOrigin/comments/154vtwy/getting_ads_on_youtube/jsvg1i0/

- https://www.reddit.com/r/uBlockOrigin/comments/154vtwy/getting_ads_on_youtube/jswd0iq/

---

Other cases:

- https://github.com/uBlockOrigin/uAssets/issues/20315
- https://www.reddit.com/r/uBlockOrigin/comments/17j6ygs/youtube_antiadblock_and_ads_october_29_2023_mega/k7e9bqx/

**Comment by krystian3w** [comment]
Ideally, the advice should be expanded to include:

- changing the home page to the uBlock Origin store page, so that at the "re"start/reboot after uninstallation you don't have to search for it via Google (less trackers from Google Search).

- suggesting that it is necessary to restart/reboot the entire browser (that is, on Windows close Chromium in the taskbar next to the clock) or even restart/reboot the system when the browser process does not want to disappear from the task manager (and reset fastboot glitches on Windows 8-11 from hiberfil.sys file).



and only then reinstall the add-on.
